eigen3.sh

```
@@ -1,12 +1,16 @@
 package: Eigen3
 version: 3.4.0
 source: https://gitlab.com/libeigen/eigen.git
+prefer_system: .*
+prefer_system_check: |
+  printf '#include <Eigen/Dense>\nint main(){}\n' | \
```

Shellcheck (0.11.0)

[warning] Piping to 'printf', a command that doesn't read stdin. Wrong command or missing xargs?

(SC2216)